integration/otlp_ingestion_test.go

```diff
@@ -103,7 +103,7 @@ func testOTLPIngestion(t *testing.T, opts testOTLPIngestionOpts) {
 	metadata := []mimirpb.MetricMetadata{
 		{
 			Help: "foo",
-			Unit: "By",
+			Unit: "bytes",
```

Comment:
Note to reviewers: this is due to bug fix in upstream. The rewrite in https://github.com/prometheus/prometheus/pull/16951 exposed it.